interpreter/cling/include/cling/Interpreter/InterpreterCallbacks.h

```diff
@@ -161,6 +162,13 @@ namespace cling {
     /// for instance when throwing interpreter exceptions.
     virtual void PrintStackTrace() {}
 
+    ///\brief Cling creates the code complete consumer for its child interp.
+    virtual void CreateCodeCompleteConsumer(Interpreter* child) const;
```

Comment:
Only Interpreter is using this, and nobody is re-implementing / overriding `CreateCodeCompleteConsumer()`. Could we move this into `Interpreter` then? Maybe even as a private function, given that it's only called by `Interpreter`?